### ISSUE TITLE
Recipes not unlocked until item is obtained?

### ISSUE THREAD

**Issue by Nekohime89** [EwyBoy/CHA-S#7]
Is this intentional?
It seems strange that ingots wouldn't unlock horse armors and leather not unlock the saddle.

**Comment by thedocruby** [comment]
Yes. I agree. Possibly another requirement would be you have to have mounted a horse at least once, because it wouldn't make sense to think of a recipe tailored to a mob that you aren't familiar with.
